web/README.md

```diff
@@ -1,4 +1,113 @@
-# Provide a guide for development on local host 
-- Tech used
- - Vite and tailwind css 
- 
\ No newline at end of file
+# # Setting up and Running Locally and Contributing
+
+
+## Tech Stack
+
+- Front-end Framework: [React.js](https://react.dev/)
+- Styling: [Tailwind CSS](https://tailwindcss.com/)
+- Build Tool: [Vite](https://vitejs.dev/)
+- Version Control: [Git](https://git-scm.com/)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) (LTS version)
+- [npm](https://www.npmjs.com/) (Node.js package manager)
+- [Git](https://git-scm.com/) (Version control system software)
+
+## Additional resources
+
+- [React.Js Documentation](https://react.dev/)
+- [Tailwind Documentation](https://tailwindcss.com/docs/installation)
+- [Vite Documentation](https://vitejs.dev/guide/)
+- [Git Documentation](https://git-scm.com/doc)
+- [MDN Web Docs](https://developer.mozilla.org/en-US/) (A comprehensive resource for web developers)
+ 
+
+## To setup StarterHive Locally 
+
+### Step 1: Fork the repository
```

Comment:
no need to provide images as of now. Just provide link to guides on forking etc from the official github guide. 
Link : https://docs.github.com/en/get-started/quickstart